docker/build-common.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
since this is not necessarily a docker thing, I suggest having this somewhere else (for example `scripts/`) and renaming to something that indicate this is for building in release mode (`release-build.sh`)

Reply:
I think there's value in that.   At the very least we could run this in circle (w/o docker) to verify the builds in prs, or should volume mounts speed it up enough w/ docker - should know today.

Reply:
@mimoo this kind of _is_ a docker thing - this script generalizes build cmd line across all docker files.
I don't want to argue too much on where to put the file though -  I don't think calling it from docker or scripts makes a difference

Reply:
no biggie, I was just thinking of people who want to build a release and run it without docker : o